### DISCUSSION TITLE
GitHub是用来搞这个的吗，差不多得了嗷

### DISCUSSION THREAD

**Discussion by MyPure** [The-Run-Philosophy-Organization/run#372] [General]

**Comment by MarionNoel** [comment]
润太祖可是特大人命事故入狱的局长公子，贪二代，有钱的大老板。他大发善心，关爱新一代it耗材，安排咱们世世代代跟着他，给他做贡献。
https://github.com/MarionNoel/juzhanggongzi/discussions/
